packages/page-accounts/src/CreateAccount.spec.tsx

```diff
@@ -19,7 +19,7 @@ const WaitForApi = ({ children }: { children: React.ReactNode }): PropsWithChild
   return api.isApiReady ? (children) : null;
 };
 
-describe('Account Create', () => {
+describe('--SLOW--: Account Create', () => {
```

Comment:
Also don’t have a better idea just yet, but not sure this is a scalable solution. However, more than good enough to get it running and see it working.

Reply:
Some alternative might be to keep the slow/docker tests in separate folder. Another would be to have a distinct extension (eg.`.slow.spec.ts`). However, I prefer the option to specify type of test in test name. Seems most natural to me.